core-java-modules/core-java-string-operations-3/src/main/java/com/baeldung/tostring/StringCastUtils.java

```diff
@@ -0,0 +1,15 @@
+package com.baeldung.tostring;
+
+public class StringCastUtils {
+    public static String castToString(Object object) {
+        if (object instanceof String)
+            return (String) object;
+        return null;
+    }
+
+    public static String getToString(Object object) {
```

Comment:
Can we think of a better name for this method? getToString sounds a bit weird to me. What about something like objectToString or something a bit more meaningful? Open to ideas.

Reply:
what about - getStringRepresentation